src/utils/detectMobileDevices.ts

```diff
@@ -0,0 +1,3 @@
+export const isMobileDevice = () => /iPhone|iPad|iPod|Android/i.test(navigator.userAgent)
+
+export const isAppleMobileDevice = () => /iPhone|iPad|iPod/i.test(navigator.userAgent)
```

Comment:
I'm going to block this.
We should only use CSS media queries for responsiveness, and avoid checks based on the device.
I would rather have the download button visible on all devices.

Reply:
Yes, that'd be best 👍

Reply:
Ok, reverted to the commit you had previously approved.